src/document-creator/document-creator.mdx

```diff
@@ -177,3 +199,54 @@ The `"accept"` accepts an array of string, that are file types, which will set t
 If you want to accept all file types, remove the entire `"accept"` line from `"attachments"` object.
 
 ---
+
+## Document Storage field
+
+```json
+"documentStorage": {
+  "apiKey": "kNb15YYZ6N1zBlYd25cjj8PLgK6YAuvN9Gf7fPM1",
```

Comment:
Is this the actual ropsten api key ah? If it is let's just say we provide this endpoint for development purposes and there is no guarantee that the endpoint will remain the same plus there is "regular housekeeping" so files will get deleted after awhile - let's not say how long so that they don't rely on it

Reply:
ok, this is not the ropsten api key, since the ropsten endpoint doesn't need any api key right? 

This is just a random one i get from the oa function. should i change it to something else?

ok will add that this endpoint is for development purposes only. and that there is a regular housekeeping so files will get deleted after a while. also wont say how long.